### PR TITLE
Add coding style to contributing guide

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,6 +87,18 @@ This repository follows the [PSR-2 standard](http://www.php-fig.org/psr/psr-2/)
...
 
 
+## Coding style
+
+Apart from the standards above, we have a few more rules which we think makes programming more logical:
+
+1. Don't align key value pairs: it messes up diff (this applies to code, config, even docblocks)
+2. Use trailing commas after last array elements
+
+See these articles for reference:
```
Comment (dbu) [inline CONTRIBUTING.md:97]: the reference is only one article...

**Comment by dbu** [comment]
i agree with this.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -87,6 +87,18 @@ This repository follows the [PSR-2 standard](http://www.php-fig.org/psr/psr-2/)
 you must follow these rules.
 
 
+## Coding style
```
Comment (dbu) [inline CONTRIBUTING.md:90]: i would make this a sub-title of "Coding Standards" and call it "Additional Coding Style"

Comment (sagikazarmark) [inline CONTRIBUTING.md:90]: Good idea :+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (sagikazarmark) [inline CONTRIBUTING.md:97]: Actually I had more ideas in my mind with more reference, but I couldn't recall.

Comment (joelwurtz) [inline CONTRIBUTING.md:97]: Can we only specify PSR2 for the moment and remove these rules ? (To be in appliance with the .php_cs file)

After that everyone can do a pull request with a change to the .php_cs file and the documentation

Comment (sagikazarmark) [inline CONTRIBUTING.md:97]: Actually I want to close this PR without merging.

Comment (joelwurtz) [inline CONTRIBUTING.md:97]: May be worth to have the deciding worklfow for new rules in the documentation WDYT ?

Comment (sagikazarmark) [inline CONTRIBUTING.md:97]: Don't think so. I don't expect too much contribution from the community on this front.

Comment (joelwurtz) [inline CONTRIBUTING.md:97]: Good for me then.

**Comment by sagikazarmark** [comment]
Solved in #3

**Pull request closed without merging**